test/child/ChildERC721.test.js

```diff
@@ -30,7 +30,7 @@ contract('ChildERC721', (accounts) => {
     })
 
     it('Token should not exist before deposit', async() => {
-      await expectRevert(contracts.dummyERC721.ownerOf(tokenId), 'ERC721: owner query for nonexistent token')
+      await expectRevert(contracts.dummyERC721.ownerOf(tokenId), 'value out of range')
```

Comment:
So we should expect these tests to fail locally? And how did the tests pass on CI in previous PRs?